saml-identity-provider/src/main/java/se/swedenconnect/spring/saml/idp/config/configurers/Saml2AuthnRequestAuthenticationProviderConfigurer.java

```diff
@@ -340,4 +367,23 @@ protected static List<RequestedAttributeProcessor> createDefaultRequestedAttribu
     return processors;
   }
 
+  // For handling the configuration of several service provider filters ...
+  private static class MultipleServiceProviderFilter implements Saml2ServiceProviderFilter {
+
+    private final Saml2ServiceProviderFilter existing;
```

Comment:
Could be a list of `Saml2ServiceProviderFilter` to allow an arbitrary amount of filters.
Since `Saml2ServiceProviderFilter` is a predicate we should be able to `list.stream().reduce(Predicate::and).orElse(x->true).test(entityDescriptor);` to combine them.